hack/.ci/run-e2e-gke.sh

```diff
@@ -0,0 +1,110 @@
+}
+
+function handle-exit {
+  gather-artifacts || "Error gathering artifacts" > /dev/stderr
+}
+
+trap handle-exit EXIT
+
+# Allow admin to use ephemeralcontainers
+kubectl_create -f - <<EOF
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: scylladb-e2e:hotfixes
+    labels:
+      rbac.authorization.k8s.io/aggregate-to-admin: "true"
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - pods/ephemeralcontainers
+    verbs:
+    - patch
+EOF
+
+kubectl apply --server-side -f ./pkg/api/scylla/v1alpha1/scylla.scylladb.com_nodeconfigs.yaml
+kubectl wait --for condition=established crd/nodeconfigs.scylla.scylladb.com
+
+if [[ "${SO_DISABLE_NODECONFIG}" == "true" ]]; then
+  echo "Skipping NodeConfig creation"
+else
+  kubectl_create -f ./hack/.ci/manifests/cluster/nodeconfig.yaml
+  kubectl_create -n local-csi-driver -f ./hack/.ci/manifests/namespaces/local-csi-driver/
+fi
+
+SCYLLA_OPERATOR_FEATURE_GATES='AllAlpha=true,AllBeta=true'
+export SCYLLA_OPERATOR_FEATURE_GATES
+REENTRANT=true timeout -v 10m ./hack/ci-deploy.sh "${SO_IMAGE}"
+# Raise loglevel in CI.
+# TODO: Replace it with ScyllaOperatorConfig field when available.
+kubectl -n scylla-operator patch --field-manager="${field_manager}" deployment/scylla-operator --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--loglevel=4"}]'
+kubectl -n scylla-operator rollout status deployment/scylla-operator
+
+kubectl -n scylla-manager patch --field-manager="${field_manager}" deployment/scylla-manager-controller --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--loglevel=4"}]'
+kubectl -n scylla-manager rollout status deployment/scylla-manager-controller
+
+ingress_address="$( kubectl -n haproxy-ingress get svc haproxy-ingress --template='{{ .spec.clusterIP }}' )"
+
+kubectl create namespace e2e --dry-run=client -o yaml | kubectl_create -f -
+kubectl create clusterrolebinding e2e --clusterrole=cluster-admin --serviceaccount=e2e:default --dry-run=client -o yaml | kubectl_create -f -
+kubectl create -n e2e pdb my-pdb --selector='app=e2e' --min-available=1 --dry-run=client -o yaml | kubectl_create -f -
+
+kubectl -n e2e run --restart=Never --image="${SO_IMAGE}" --labels='app=e2e' --command=true e2e -- bash -euExo pipefail -O inherit_errexit -c "mkdir /tmp/artifacts && scylla-operator-tests run '${SO_SUITE}' --parallelism=60 --loglevel=2 --color=false --artifacts-dir=/tmp/artifacts --feature-gates='${SCYLLA_OPERATOR_FEATURE_GATES}' --override-ingress-address='${ingress_address}' && touch /tmp/done && until [[ -f '/tmp/exit' ]]; do sleep 1; done"
+kubectl -n e2e wait --for=condition=Ready pod/e2e
+
+# Setup artifacts transfer when finished and unblock the e2e pod when done.
+(
+  kubectl -n e2e exec pod/e2e -- bash -euEo pipefail -O inherit_errexit -c "until [[ -f /tmp/done ]]; do sleep 1; done; ls -l /tmp/artifacts"
```

Comment:
`/tmp/artifacts` could be extracted to variable, as it's used in multiple places

Reply:
I don't think that's worth it, especially with the nested shells and reasoning about where it gets evaluated.

Reply:
but the 60 is a leftover, thx

Reply:
done